x-pack/plugins/canvas/public/components/sidebar/sidebar.scss

```diff
@@ -31,12 +31,68 @@
   &--isEmpty {
     border-bottom: none;
   }
+
+  .canvasSidebar__expandable:last-child {
+    .canvasSidebar__accordion {
+      margin-bottom: (-$euiSizeS);
+    }
+
+    .canvasSidebar__accordion:after {
+      content: none;
+    }
+
+    .canvasSidebar__accordion.euiAccordion-isOpen:after {
+      display: none;
+    }
+  }
 }
 
 .canvasSidebar__panel-noMinWidth .euiButton {
   min-width: 0;
 }
 
+.canvasSidebar__expandable + .canvasSidebar__expandable {
+  margin-top: 0;
+
+  .canvasSidebar__accordion:before {
+    display: none;
+  }
+}
+
+.canvasSidebar__accordion {
+  padding: $euiSizeM;
+  margin: 0 (-$euiSizeM);
+  background: $euiColorLightestShade;
+  position: relative;
+
+  &.euiAccordion-isOpen {
+    background: transparent;
+  }
+
+  &:before,
+  &:after {
+    content: '';
+    height: 1px;
+    position: absolute;
+    left: 0;
+    width: 100%;
+    background: $euiColorLightShade;
+  }
+
+  &:before {
+    top: 0;
+  }
+
+  &:after {
+    bottom: 0;
+  }
+}
+
+.canvasSidebar__accordionContent {
+  padding-top: $euiSize;
+  padding-left: $euiSizeL;
```

Comment:
I it was previously suggested to change this to the `$euiSizeL` variable, but I'm going to swoop in and suggest that we actually change it again :)

The whitespace required to line up with the beginning of the accordion title's text is actually 28px. `$euiSizeL` is 24px, making it about 4px off from being properly aligned. Instead, we should probably switch to `padding-left: euiSizeXS + euiSizeS + euiSize;`. This will result in a value of 28px and represent the individual variable values for the accordion icon's width and its left and right margining.